src/pfe/portal/routes/imagePushRegistry.route.js

```diff
@@ -38,33 +58,31 @@ router.get('/api/v1/registry', async function (req, res) {
 /**
  * API Function to test and set the Deployment Registry
  */
-router.post('/api/v1/registry', validateReq, async function (req, res) {
+router.post('/api/v1/imagepushregistry', validateReq, async function (req, res) {
   let retval;
   try {
     let user = req.cw_user;
-    log.debug(`POST /api/v1/registry called`);
-    const deploymentRegistry = req.sanitizeBody('deploymentRegistry');
+    log.debug(`POST /api/v1/imagepushregistry called`);
+    const address = req.sanitizeBody('address');
+    const namespace = req.sanitizeBody('namespace');
     const operation = req.sanitizeBody('operation');
-    if (!deploymentRegistry) {
-      const msg = "Missing required parameter, deploymentRegistry is required to be provided.";
-      const data = { "statusCode": 400, "deploymentRegistryTest": false, "msg": msg}
-      res.status(400).send(data);
-    }
+
     // The validateReq middleware will throw an error if operation is not test or set,
     // but it is optional and defaults to test.
     if (operation === undefined || operation === 'test') {
-      log.debug(`Testing deployment registry: ${deploymentRegistry}`);
-      retval = await user.testDeploymentRegistry(deploymentRegistry);
+      log.debug(`Testing image push registry: ${address}/${namespace}`);
```

Comment:
Do we need to make sure address does not ends with / ? Alternative is to control that when we read in address.

Reply:
yes we're doing that when we read in address `imagePushRegistry = workspaceSettingsInfo.registryAddress.replace(/\/\s*$/, "") + "/" + workspaceSettingsInfo.registryNamespace.trim();`